docs/docs/configuration/analyzer-diagnostics/RMG066.mdx

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_label: RMG066
+description: 'Mapperly analyzer diagnostic RMG066 — No members are mapped in an object mapping'
+---
+
+# RMG066 — No members are mapped in an object mapping
+
+If Mapperly maps to an object without mapping any member, `RMG066` is emitted.
+
+The generated implementation part of the mapping method usually looks like this:
+
+```csharp
+public partial Target Map(Source source)
+{
+    var target = new Target();
+    return target;
+}
+```
+
+Usually this is not expected.
+If this is expected it can be suppressed for a mapping by applying `[ExpectMemberMappings(false)]`
```

Comment:
Does it make sense to create a new attribute for this, increasing the API surface?

Users could simply ignore this warning by suppressing it, couldn't they?